ontology/yaml/resources/HVAC/entity_types/DWST.yaml

```diff
@@ -72,3 +72,16 @@ DWST_WDT:
   implements:
   - DWST
   - WDT
+
+DWST_CPC_RMM_SS_REFPM_EPMX2_HRCTMPSM:
```

Comment:
The heat pumps and circulation pumps should be modeled separately from the DWST.